aten/src/ATen/native/hammerblade/Empty.cpp

```diff
@@ -58,4 +43,10 @@ Tensor empty_hb(IntArrayRef size, const TensorOptions& options, c10::optional<c1
   return tensor;
 }
 
+Tensor empty_strided_hb(IntArrayRef size, IntArrayRef stride, const TensorOptions& options) {
+  auto t = at::native::empty_hb({0}, options);
+  at::native::resize_impl_hb_(t.unsafeGetTensorImpl(), size, stride);
+  return t;
+}
```

Comment:
Just curious... is this to create empty tensors with strides of our choice?

Reply:
Good question. I have to admit that I'm not 100% sure what it does, but I think you are right